src/sfizz/ADSREnvelope.cpp

```diff
@@ -122,7 +122,8 @@ void ADSREnvelope::getBlock(absl::Span<Float> output) noexcept
                 break;
             }
             while (count < size) {
-                currentValue = std::max(sustain, currentValue + transitionDelta);
+                if (currentValue > sustain)
```

Comment:
Can you move the branch out the loop? that is
```
if()
  while () { .. }
else
  while () { ... }
```

Reply:
I wonder if actually it wouldn't just be
```c++
if (currentValue > sustain)
    currentValue += transitionDelta;
```

Reply:
(Note that there are branches in std::max, at least in libstdc++)